components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -7823,6 +7823,37 @@ def setInvertedAxis(self, inverted):
         (1, False, True): 'b',  # signed char
         }
 
+    @assert_pixels
+    def getHistogram(self, channels, binCount, globalRange=True,
```

Comment:
maybe time to unify method name with other part of the python code i.e not using upper case in the method name.
Changing the existing ones will be a major effort
but maybe the new ones. 
I realise that it will not "match" the current pattern of the file but we need to start at some point